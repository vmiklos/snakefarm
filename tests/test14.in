debug 1
out 1
time 100
load test14.map
addplayer 1
addsnake 1 1 0 1 1 1
18 0
17 0
endsnake
addplayer 2
addsnake 2 2 3 1 1 0
19 0
12 0
endsnake
comment # a masodik kigyonak meg kell halnia, az elsonek sertetlennek maradnia, es miutan az elso lepett, a masodik mar nem is tud
step
exit