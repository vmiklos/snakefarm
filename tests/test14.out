Event MapLoad test14.map
# a masodik kigyonak meg kell halnia, az elsonek sertetlennek maradnia, es miutan az elso lepett, a masodik mar nem is tud
Event Step 1
StepEvent Snake_Snake 2
StepEvent SnakeDie 2
StepEvent SnakeModeNormal
End Step
Event Exit